x/go/math/math.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2024 Synnax Labs, Inc.
+//
+// Use of this software is governed by the Business Source License included in the file
+// licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with the Business Source
+// License, use of this software will be governed by the Apache License, Version 2.0,
+// included in the file licenses/APL.txt.
+
+// math is a package that extends math functionality.
+
+package math
+
+// IntPow returns the integer n to the m power. IntPow requires m to be a

```

Comment:
Not sure how applicable this is, but there's a nice way to do power in log(m) time https://en.wikipedia.org/wiki/Exponentiation_by_squaring – might be useful if m can be large